src/ashley/api/serializers.py

```diff
@@ -0,0 +1,45 @@
+"""Serializer for ashley api."""
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+from ashley.context_mixins import get_current_lti_session
+from ashley.defaults import (
+    _FORUM_ROLE_INSTRUCTOR,
+    _FORUM_ROLE_MODERATOR,
+    _FORUM_ROLE_STUDENT,
+)
+
+User = get_user_model()
+
+
+class UserSerializer(serializers.ModelSerializer):
+    """Serializer for User model."""
+
+    role = serializers.SerializerMethodField()
+
+    class Meta:
+        model = User
+        fields = ["id", "public_username", "role"]
+        read_only_fields = ["public_username", "role"]
+
+    def get_role(self, obj):
+        """Describes the role of the user"""
+        context = get_current_lti_session(self.context.get("request"))
+        if obj.is_active:
+            if obj.groups.filter(
+                name=context.get_group_role_name(_FORUM_ROLE_MODERATOR)
+            ).count() == 0 and obj.groups.filter(
+                name=context.get_group_role_name(_FORUM_ROLE_STUDENT)
+            ):
+                return _FORUM_ROLE_STUDENT
+            if obj.groups.filter(
```

Comment:
It is not possible to do it in one queryset ? 

```suggestion
            if obj.groups.filter(name=context.get_group_role_name(_FORUM_ROLE_STUDENT), name=context.get_group_role_name(_FORUM_ROLE_STUDENT))
```